Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -7929,6 +7933,7 @@ void WebPageProxy::didReceiveEvent(WebEventType eventType, bool handled)
         bool canProcessMoreKeyEvents = !m_keyEventQueue.isEmpty();
         if (canProcessMoreKeyEvents) {
             LOG(KeyHandling, " UI process: sent keyEvent from didReceiveEvent");
+            m_process->userInitiatedActivity(event.authorizationToken());
```

Comment:
It looks weird to be calling a getter and expect side effects here.

I think we should either rename this function or split it into a getter & setter.